optuna_dashboard/ts/components/PreferentialTrials.tsx

```diff
@@ -12,41 +24,119 @@ const PreferentialTrial: FC<{
+          minHeight: trialHeight,
+          margin: theme.spacing(2),
+        }}
+      />
+    )
   }
 
   return (
-    <Box sx={{ width: trialWidth, padding: theme.spacing(2, 2, 0, 2) }}>
-      <Typography
-        variant="h4"
-        sx={{
-          marginBottom: theme.spacing(2),
-          fontWeight: theme.typography.fontWeightBold,
-        }}
-      >
-        Trial {trial.number} (trial_id={trial.trial_id})
-      </Typography>
-      <Button
-        variant="outlined"
+    <Card
+      sx={{
+        width: trialWidth,
+        minHeight: trialHeight,
+        position: "relative",
+        margin: theme.spacing(2),
+        padding: 0,
+      }}
+    >
+      <CardActions>
+        <IconButton
+          sx={{
+            marginLeft: "auto",
+          }}
+          onClick={() => setDetailShown(true)}
+          aria-label="show detail"
+        >
+          <OpenInFullIcon />
+        </IconButton>
+      </CardActions>
+      <CardContent
+        aria-label="trial-button"
         onClick={() => {
           hideTrial()
           const best_trials = studyDetail.best_trials
             .map((t) => t.number)
             .filter((t) => t !== trial.number)
           action.updatePreference(trial.study_id, best_trials, [trial.number])
         }}
+        sx={{
+          padding: 0,
+          position: "relative",
+          overflow: "hidden",
+          "::before": {
+            content: '""',
+            position: "absolute",
+            top: 0,
+            left: 0,
+            width: "100%",
+            height: "100%",
+            backgroundColor: theme.palette.mode === "dark" ? "white" : "black",
+            opacity: 0,
+            zIndex: 1,
+            transition: "opacity 0.3s ease-out",
+          },
+          ":hover::before": {
+            opacity: 0.1,
+          },
+        }}
       >
-        Worst
-      </Button>
-      <TrialNote
-        studyId={trial.study_id}
-        trialId={trial.trial_id}
-        latestNote={trial.note}
-        cardSx={{ marginBottom: theme.spacing(2) }}
-      />
-    </Box>
+        <Box
+          sx={{
+            padding: theme.spacing(2),
+            position: "relative",
+          }}
+        >
+          <MarkdownRenderer body={trial.note.body} />
+        </Box>
+
+        <ClearIcon
+          sx={{
+            position: "absolute",
+            width: "100%",
+            height: "100%",
+            top: 0,
+            left: 0,
+            color: "red",
+            fontSize: trialWidth / 2,
+            opacity: 0,
+            transition: "opacity 0.3s ease-out",
+            zIndex: 1,
+            ":hover": {
+              opacity: 0.5,
+            },
+          }}
+        />
+      </CardContent>
+      <Modal open={detailShown} onClose={() => setDetailShown(false)}>
+        <Box
+          sx={{
+            position: "absolute",
+            width: "70%",
+            left: "15%",
+            top: theme.spacing(8),
```

Comment:
```suggestion
            position: "absolute",
            top: 0,
            left: 0,
            right: 0,
            bottom: 0,
            width: "80%",
            maxHeight: "90%",
            margin: "auto",
```

How about positioning to the center like this?

<img width="2266" alt="Screenshot 2023-08-28 16 27 00" src="https://github.com/optuna/optuna-dashboard/assets/5564044/76e6a29a-3edb-4021-a1d0-90fb21db753d">